src/Type/Php/JsonThrowTypeExtension.php

```diff
@@ -52,11 +52,20 @@ public function getThrowTypeFromFunctionCall(
 	): ?Type
 	{
 		$argumentPosition = $this->argumentPositions[$functionReflection->getName()];
-		if (!isset($functionCall->getArgs()[$argumentPosition])) {
+
+		$optionsExpr = null;
+		$args = $functionCall->getArgs();
+		foreach ($args as $i => $arg) {
+			if ($arg->name === null && $i === $argumentPosition || $arg->name !== null && $arg->name->toString() === 'flags') {
```

Comment:
Please use disambiguationg parentheses so that we know how this condition is actually evaluated.